org.eclipse.xtext.maven.plugin/src/test/java/org/eclipse/xtext/maven/plugin/XtextGeneratorIT.java

```diff
@@ -49,6 +49,7 @@ public class XtextGeneratorIT {
 	static public void setUpOnce() throws IOException, VerificationException {
 		testDir = extractTestRoot();
 		if (fork) {
+			System.out.println("maven.home=" + System.getProperty("maven.home"));
```

Comment:
Do you think this should stay here?

Reply:
Yes, that would be intentional: it will always show the value of maven.home, even when not set. This should help debug possible problems, even when running locally. Remember that once the verifier object is created, it will swallow System.out, System.err, so it's better to know this information before creating it.

Reply:
Maybe make it more useful and print the value of "maven.home" here and after the dance that happens in the lines below

Reply:
@szarnekow as I said, after a `Verifier` object is created, System.out and System.err are not directed to the console anymore; anything printed after the next line will be swallowed.
What I could do is save `System.out` in advance, what do you think?
And then print the result: either the executable or the result of `findMaven`.

Reply:
> What I could do is save System.out in advance, what do you think?

Yes, I'd probably save it as a field on the class to have it available for other logging, too. WDYT?

Reply:
(static field, that is)

Reply:
@szarnekow done here: https://github.com/eclipse/xtext/pull/2950/commits/83ecc709e46aae6a1b2e4e17ac99d85986ec6a31